src/App.js

```diff
@@ -92,7 +92,9 @@ function App() {
                     <Route
                       exact
                       path={page.pageLink}
-                      component={page.view}
+                      render={({match}) => (
+                        <page.view key={match.params.stateCode || index} />
```

Comment:
What does the `|| index` do? 🤔

Reply:
key prop forces the page re-render while choosing the different state from the breadcrumb dropdown

Reply:
Oh, that's pretty cool actually! I'm wondering why <Link> doesn't do the re-render though, any idea?